python_scripts/config/defaults.py

```diff
@@ -0,0 +1,7 @@
+# database configurations
+
+DB_HOST = 'localhost'
+DB_PORT = 3306
+DB_USER = 'root'
+DB_PWD = ''
+DB_NAME = 'biosero_uat'
```

Comment:
For localhost: biosero_development?

Reply:
yeah I want to change this name anyway so it's not 'biosero', but yes it's dev you are correct. It's in all the create table and other queries too.